src/main.rs

```diff
@@ -36,6 +38,13 @@ fn main() {
                     .help("Just print the hex values, not colour previews")
                     .takes_value(false)
             )
+            .arg(
+                Arg::with_name("output")
```

Comment:
The help text for this argument as written is going to be something like:

```
dominant_colours [PATH] --output=<output>
```

But variables in command-line help text are typically uppercase, i.e.

```
dominant_colours [PATH] --output=<OUTPUT>
```

Whatever we end up calling this argument, let's make the variable uppercase.